### PR TITLE
CI: Run SBML test suite on pull request only if AMICI SBML import rou…

### PR DESCRIPTION
**Pull request by dweindl**
…tines are changed (#1054)

### CODE REVIEW

**Comment by dweindl** [comment]
@FFroehlich : Should now trigger sbml test suite everytime SBML import gets modified. However, not automatically on PR to master (not supported). Merge/push to develop will still trigger always, so that shouldn't be too much of a problem.

**Comment by paulstapor** [comment]
Nice, thanks!

File: .github/workflows/sbml-semantic-test-suite.yml [hunk 1]
```diff
@@ -5,8 +5,9 @@ on:
       - develop
       - master
   pull_request:
-    branches:
-      - master
+    paths:
+      - python/amici/sbml_import.py
+      - python/amici/ode_export.py
```
Comment (FFroehlich) [inline .github/workflows/sbml-semantic-test-suite.yml:10]: add `scripts/run-SBMLTestsuite.sh` and `tests/testSBMLSuite.py`?

Comment (dweindl) [reply .github/workflows/sbml-semantic-test-suite.yml:10]: Sounds reasonable

**Pull request merged**